charts/opentelemetry-collector/README.md

```diff
@@ -0,0 +1,107 @@
+# OpenTelemetry Collector Helm Chart
+
+The helm chart installs [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) 
+in kubernetes cluster.
+
+## Prerequisites
+
+- Helm 3.0+
+
+## Installing the Chart
+
+Add OpenTelemetry Helm repository:
+
+```console
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+```
+
+To install the chart with the release name my-opentelemetry-collector, run the following command:
+
+```console
+helm install my-opentelemetry-collector open-telemetry/opentelemetry-collector
+```
+
+## Configuration
+
+### Default configuration
+
+By default this chart will deploy an OpenTelemetry Collector as daemonset with two pipelines (traces and metrics) 
+and logging exporter enabled by default. Besides daemonset (agent), it can be also installed as standalone deployment. 
+Both modes can be enabled together, in that case metrics and traces will be flowing from agents to standalone collectors.
+
+*Example*: Install collector as a standalone deployment, and do not run it as an agent.
+```yaml
+agentCollector:
+  enabled: false
+standaloneCollector:
+  enabled: true
+```
+
+By default collector has the following receivers enabled:
+- **metrics**: OTLP and prometheus. Prometheus is configured only for scraping collector's own metrics.
+- **traces**: zipkin and jaeger (thrift and grpc).
```

Comment:
Why don't we also enable OTLP receiver for traces?

Reply:
it is enabled, missed to mention it in the docs. fixed